docs-chef-io/content/inspec/resources/firewalld.md

```diff
@@ -40,10 +40,24 @@ Use the where clause to test open interfaces, sources, and services in active zo
       its('interfaces') { should cmp ['enp0s3', 'eno2'] }
       its('sources') { should cmp ['192.168.1.0/24', '192.168.1.2'] }
       its('services') { should cmp ['ssh', 'icmp'] }
+      its('target') { should cmp ['default'] }
     end
 
 ## Properties
 
+- `interfaces`
+- `sources`
+- `services`
+- `target`
+- `ports`
+- `protocols`
+- `forward_ports`
+- `source_ports`
+- `icmp_blocks`
+- `rich_rules`
+
+## Property Examples
```

Comment:
We are removing `Property Examples` section from the docs are going to list out all `Property Examples` under the same head `Property`. So doing that change will help reduce the rework.

Reply:
@Vasu1105 Removed the old `Properties` section and renamed `Property Examples` as the new `Properties` -- let me know if that works for how you want the docs to look.